src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    <ul>
+      {todos.map(todo => (
+        <li key={todo.id}>
+          <TodoInfo todo={todo} />
+        </li>
+      ))}
+    </ul>
+  </section>
```

Comment:
you dont need to put it inside 'ul' and 'li' tags because it gives unnecessary list styles (dot in front of every todo block)
```suggestion
    <section className="TodoList">
      {todos.map(todo => <TodoInfo todo={todo} key={todo.id} />)}
    </section>
```